public/data/viktorCodes.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Victor Nkwanyuo",
+    "type": "personal",
+    "bio": " Full-Stack Software Engineer && Open Source Enthusiast!",
+    "avatar": "https://github.com/viktorCodes.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/viktorCodes",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/nviktordev",
+        "icon": "twitter"
+      }
+      ,
+      {
+        "name": "Follow me on Linkedin",
```

Comment:
I recommend capitalizing the `i` here. What do you think?
```suggestion
        "name": "Follow me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question